sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py

```diff
@@ -26,9 +26,10 @@
 
```

Comment:
Could we please restructure this file so its a bit cleaner? At least group variables and executions together? Somehting like here?
https://github.com/shadow-robot/sr_interface/blob/bb86eff59f818052c25f3ae72964d45ed454317b/sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py